### ISSUE TITLE
"file with no instructions" error when committing a container

### ISSUE THREAD

**Issue by mmstick** [fussybeaver/bollard#37]
Any idea why this might happen?

```
API queried with a bad parameter: {"message":"file with no instructions."}
```

```rust
pub fn save(&mut self, container: &str, repo: &str) -> Result<(), Error> {
    self.validate_container_exists(container)?;

    let options = CommitContainerOptions {
        container: container.to_owned(),
        author: "tensorman".into(),
        comment: "automated image creation by tensorman".into(),
        pause: true,
        repo: ["tensorman/", repo].concat(),
        ..Default::default()
    };

    let config = ContainerConfig::<String> { ..Default::default() };
    let future = self.docker.commit_container(options, config);

    self.tokio.block_on(future).map_err(|failure| failure.compat()).map_err(|source| {
        Error::Commit { container: container.into(), repo: repo.into(), source }
    })?;

    Ok(())
}
```

**Comment by fussybeaver** [comment]
Thanks for reporting.. 

It looks like the `commit` parameter in `CommitContainer` can't be fed with an empty string if you don't want it to be used. 

I've committed a fix in bollard in #38

**Comment by fussybeaver** [comment]
Fixed in #40
